content/docs/user-guide/assets/asset-processor/debugging.md

```diff
@@ -122,3 +122,107 @@ If you're a game artist and you're having issues running Asset Processor, the is
+
+```cmd
+<path to asset processor>/AssetProcessorBatch.exe --debugOutput
+```
+
+For example:
+
+```cmd
+D:\o3de\build\bin\profile\AssetProcessor.exe --debugOutput
+```
+
+The debug output flag can be set in the Asset Processor GUI using the Tools | Debug Output check box. When this check box is active, debug output files will be found in the cache folder for the scene files.
+
+
+![Asset Processor UI Debug Output](/images/user-guide/assets/asset-processor/debug_output.png)
+
+{{< note >}}
+After turning on the debug output flag, the asset needs to be reprocessed to output the debug files.
+{{< /note >}}
+
+#### Debug Output: Scene Graph
+
+The scene graph debug output files are stored next to the default `.azmodel` file. For example, for a source file in `D:\o3de\my_project\assets\test.fbx` for the PC platform, the Cache folder should have (at least) these files:
+
+```cmd
+D:\o3de\my_project\Cache\pc\assets\test.azmodel
+D:\o3de\my_project\Cache\pc\assets\test.dbgsg
+D:\o3de\my_project\Cache\pc\assets\test.dbgsg.xml
+```
+
+The `.dbgsg` and `.dbgsg.xml` files are the debug scene graph files where the former is a flat list of debug information per node and the latter is an XML representation of the debug information of the nodes in the file.
+
+The debug output lists the node name, the node path, and the node type. The name is the text label the author assigned to the node. The node path is the dotted notation that leads to the node from the root node. The node type stores specific data for that node such as mesh data, transform data, or custom property data. 
+
+The Mesh Data stores information about the mesh such as the count of positions, normals, face list, and face material IDs. The Transform Data stores information about the matrix translation, scale, and rotation. The Material Data stores information such as its name and physical base rendering properties. The Custom Property Data stores the user defined properties in key-value pairs.
+
+#### Debug Output: Scene Manifest
+
+The scene pipeline logic can be altered using Python scripts or C++ code to update the scene manifest. To determine how the logic affected the scene manifest rules a team can turn on the debug output flag and find the 1.assetinfo.dbg1 file in the Cache folder.
+
+For example:
+
+```cmd
+D:\o3de\my_project\Cache\pc\assets\test.azmodel
+D:\o3de\my_project\Cache\pc\assets\test.assetinfo.dbg
+```
+
+The `.assetinfo.dbg` file is a file representation of the scene manifest that was in memory when the scene builder processed the scene graph. Each rule starts with the `“$type”` key and lists the rule by both GUID and name such as `"{07B356B7-3635-40B5-878A-FAC4EFD5AD86} MeshGroup"`. 
+
+The MeshGroup is an example of a rule where it creates an `.azmodel` product file named (via "name"), includes the mesh node paths in the "selectedNodes" array, and excludes the node paths in the "unselectedNodes" array.

```

Comment:
Awkward phrase. Please correct my suggestion if the rewording is not accurate.
Key-value fields should have code-style formatting.
Also, make sure you are only using double-quotes for the keys and values if the double-quotes actually appear in the file.
```suggestion
The MeshGroup is an example of a rule where it creates an `.azmodel` product file that is named using the `"name"` field, includes the mesh node paths in the `"selectedNodes"` array, and excludes the node paths in the `"unselectedNodes"` array.

```